src/components/NewsletterView.js

```diff
@@ -23,17 +30,35 @@ class NewsletterView extends React.Component {
     if (!issue) {
       return null;
     }
+    this.props.history.listen(stuff =>
+      this.issueRef.current ? this.issueRef.current.focus() : null
+    );
+
+    const date = new Date('01/' + issue.Date); // Add 01/ to be valid date, e.g. 01/01/2019
+    const dateFormatted = new Intl.DateTimeFormat('en-US', {
+      // Request only month and year, since the day was fake
+      month: 'short',
+      year: 'numeric'
+    }).format(date);
 
     return (
-      <div className="Issue-view">
+      <main className="Issue-view">
         <h2>
-          <a href={issue.Link} target="_blank" rel="noopener noreferrer">
-            Phaser World Issue {issue.Issue}
+          <a
+            href={issue.Link}
+            target="_blank"
+            rel="noopener noreferrer"
+            ref={this.issueRef}
```

Comment:
What benefit do we get by adding a ref to the anchor element?

Reply:
I think it was my attempt at making sure the focus changed in a meaningful way when you choose an issue in the menu. Should make navigating with keyboard/screen reader a little more pleasant.